postprocessing/split_ncvars/split_ncvars.pl

```diff
@@ -140,7 +140,7 @@
     if (!@varlist) {
         if (!$Opt{STATIC}) {
 	    print tailname($list_ncvars)." -t0123 $file\n" if $Opt{VERBOSE} > 1;
```

Comment:
Can you change this to -t01234 as well?

Reply:
Done. Great catch, thanks Will. (Sorry for the long delay, I didn't catch the github notification)